src/main/validation/amlBodyMembershipNumber.ts

```diff
@@ -0,0 +1,11 @@
+import { body, ValidationChain } from "express-validator";
+
+const amlBodyMembershipNumberControllerValidator = () : ValidationChain[] => {
+    const amlBodyMembershipNumberErrors : ValidationChain[] = [];
+    for (let i = 1; i < 26; i++) {
```

Comment:
I think number 26 here is the number of AML supervisory bodies? Can we declare it as constant or configurable please? Instead of hardcoding it in for condition.

Reply:
Now actually we can get the size of the array from AML supervisory bodies enum